dev/src/main/java/org/eclipse/codewind/intellij/ui/toolwindow/UpdateHandler.java

```diff
@@ -104,6 +109,23 @@ public void removeApplication(CodewindApplication application) {
                     }
                 }
             }
+            // Cannot add an update handler listener to the Debug tool window since we don't control when it is created.
+            // Whenever we get notified of a project removal, we'll have to find the appropriate Debug tool window
+            // and remove it.
+            Project[] openProjects = ProjectManager.getInstance().getOpenProjects();
+            Project currentProject = null;
+            for (Project project : openProjects) {
+                @SystemIndependent String basePath = project.getBasePath();
+                if (application.fullLocalPath.toAbsolutePath().toString().equals(basePath)) {
```

Comment:
This test won't work correctly on Windows.

Reply:
See new changes to FileUtil